source/usage-examples/updateMany.txt

```diff
@@ -30,6 +30,24 @@ Example
 
 .. include:: /includes/connect-guide-note.rst
 
-.. literalinclude:: /code-snippets/usage-examples/updateMany.js
-  :language: javascript
-  :linenos:
+.. tabs::
+
+   .. tab:: JavaScript
+      :tabid: javascript
+ 
+      .. literalinclude:: /code-snippets/usage-examples/updateMany.js
+         :language: javascript
+         :linenos:
+ 
+   .. tab:: TypeScript
+      :tabid: typescript
+ 
+      .. literalinclude:: /code-snippets/usage-examples/updateMany.ts
+         :language: typescript
+         :linenos:
+
+If you run the example above, you should see output that resembles the following:
```

Comment:
I feel like I've commented before that "resembles the following:" feels stuffy and could be simplified:

>...output like:

>...output similar to:

>...output resembling:

Reply:
I agree resembles the following feels stuffy. I think we accept both forms in drivers docs, but as I also prefer this phrasing I'll change it to "output like".